### ISSUE TITLE
code for predicting risk scores

### ISSUE THREAD

**Issue by sjjadsa** [ajv012/SurvPath#19]
could u please release the code for predicting risk scores,especially using the model from each and predict the correspoding validation set.thanks
> We do not provide the code for drawing KM curves as it is pretty standard code from [lifelines](https://lifelines.readthedocs.io/en/latest/fitters/univariate/KaplanMeierFitter.html). However, here's an overview of the process. We first use the model from each fold post training to predict the risk scores on the validation set for that fold. Then we concat the risk scores for all of the validation folds, giving us the risk scores for the entire dataset. We then use lifelines to fit a KM fitter and plot in 2 risk groups (high and low). The same package is used to perform the logrank statistical test between risk groups as well. 
> 
> 
> 
> Please let us know if something is unclear and we would be happy to help! 

 _Originally posted by @ajv012 in [#15](https://github.com/mahmoodlab/SurvPath/issues/15#issuecomment-2546910072)_

**Comment by guillaumejaume** [comment]
From the logits, the risks are calculated as:

https://github.com/mahmoodlab/SurvPath/blob/main/utils/core_utils.py#L409

**Comment by sjjadsa** [comment]
thank u
